layouts/robots.txt

```diff
@@ -6,6 +6,10 @@ Disallow: /v1.1/
 Disallow: /404/
 Disallow: /404.html
 
+{{ if strings.Contains .Site.BaseURL ".netlify.app" -}}
+# Don't crawl Netlify staging sites
+Disallow: /
+{{ end }}
```

Comment:
I'm OK with this. We might want / need to edit `layouts/index.headers` too - have a look for `X-Robots-Tag:` headers.

Reply:
wow, I don't know that file before!